app/src/routes/register/register.vue

```diff
@@ -0,0 +1,112 @@
+<script setup lang="ts">
+import { useAppStore } from '@directus/stores';
+import { useHead } from '@unhead/vue';
+import { computed, ref } from 'vue';
+import { useI18n } from 'vue-i18n';
+import RegisterForm from './register-form.vue';
+
+withDefaults(
+	defineProps<{
+		logoutReason?: string | null;
+	}>(),
+	{
+		logoutReason: null,
+	},
+);
+
+const { t, te } = useI18n();
+const appStore = useAppStore();
+const authenticated = computed(() => appStore.authenticated);
+const wasSuccessful = ref(false);
+
+useHead({
+	title: t('register'),
+});
+</script>
+
+<template>
+	<public-view>
+		<div class="header">
+			<h1 class="type-title">{{ wasSuccessful ? t('registration_successful_headline') : t('register') }}</h1>
+		</div>
+
+		<div v-if="wasSuccessful" class="after-success">
+			<div v-md="t('registration_successful_note')"></div>
```

Comment:
Does it make sense to have two different notices depending on wether the verification email is actually enabled or not. UX wise it feels a bit weird and I could see users waiting for an email to arrive with the notice being worded like this.